app/controllers/account_users_controller.rb

```diff
@@ -31,7 +31,8 @@ def create
     @account_user.created_by = session_user.id
 
     if @account_user.save
-      LogEvent.log(@account_user, :create, current_user)
+      LogEvent.log(
+        @account_user, :create, @account_user.created_at, current_user)
```

Comment:
I get why we may want to be able to log an event to the past to deal with importing, but I'm not sure I see the value in requiring passing a timestamp in every time. What if we made the timestamp an optional keyword argument that defaults to `Time.current`?